BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -234,48 +225,66 @@ public final class ConvivaAnalytics: NSObject {
      */
     public func pauseTracking() {
         // AdStart is the preferred way to pause tracking according to conviva.
-        client.adStart(sessionKey,
-                       adStream: .ADSTREAM_SEPARATE,
-                       adPlayer: .ADPLAYER_SEPARATE,
-                       adPosition: .ADPOSITION_PREROLL)
-        client.detachPlayer(sessionKey)
+        videoAnalytics?.reportAdBreakStarted(AdPlayer.ADPLAYER_SEPARATE,
+                                             adType: AdTechnology.CLIENT_SIDE,
```

Comment:
Can we make this configurable (but with `CLIENT_SIDE` as default)? There are use cases where we have server-side ad insertion but the player emits the ad events in the same way as if it would be client-side.

Reply:
Is `pauseTracking` only be used for ads? 🤔  When is this supposed to be used, for server-side ads? As otherwise the AdBreak handling would cover this anyway.